### PR TITLE
Move documentation from Wiki to GitHub

### PR DESCRIPTION
**Pull request by oleg-nenashev**
subj

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Jenkins understands the JUnit test report XML format (which is also used by TestNG). 
+When this option is configured, Jenkins can provide useful information about test results, such as trends.
+
+The plugin also provides a generic API for other unit-test publisher plugins in Jenkins. This functionality was part of the Jenkins Core until it was split out to this plugin in version in 1.577.
+
+## Configuration
+
+The JUnit publisher is configured at the job level by adding a Publish JUnit test result report post build action. The configuration parameters include:
+
+* **Test report XMLs:** Specify the path to JUnit XML files in the Ant glob syntax, such as **/build/test-reports/*.xml.
```
Comment (jglick) [inline README.md]: use backticks

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* **Test report XMLs:** Specify the path to JUnit XML files in the Ant glob syntax, such as **/build/test-reports/*.xml. 
+  Be sure not to include any non-report files into this pattern. 
+  You can specify multiple patterns of files separated by commas. 
+  The base directory of the fileset is the workspace root.
+* **Retain long standard output/error:** If checked, any standard output or error from a test suite will be retained in the test results after the build completes. 
+  (This refers only to additional messages printed to console, not to a failure stack trace). 
+  Such output is always kept if the test failed, but by default lengthy output from passing tests is truncated to save space. 
+  Check this option if you need to see every log message from even passing tests, but beware that Jenkins's memory consumption can substantially increase as a result, even if you never look at the test results!
+* **Health report amplification factor:** The amplification factor to apply to test failures when computing the test result contribution to the build health score.
```
Comment (jglick) [inline README.md]: I think this sort of detailed help is best left in `help*.html` files in the product. Will just go stale here. The online docs should focus on higher-level guidelines and suggestions.

Comment (jglick) [reply README.md]: ... but this is just a port of the Wiki page, so fine for now.

Comment (oleg-nenashev) [reply README.md]: Yeah, did not want to spend much time rewording. Subject for contributions

File: README.md (hunk 1 above)
Comment (oleg-nenashev) [reply README.md]:
```suggestion
* **Test report XMLs:** Specify the path to JUnit XML files in the Ant glob syntax, such as `**/build/test-reports/*.xml`. 
```

**Comment by olivergondza** [comment]
What would happen with the user documentation on plugin wiki? I prefer to have user-docs there...

**Comment by zbynek** [comment]
@olivergondza the first paragraph in the wiki tells users to go to the plugin site instead (I assume the plugin site is easier to scale/customize). So the plugin site is preferred as source for the users. Until now wiki was the only way to edit the content served on the plugin site, but now with https://issues.jenkins-ci.org/browse/WEBSITE-406 released you can use GitHub as the editor as well.

**Comment by oleg-nenashev** [comment]
@olivergondza @zbynek Sorry, missed the ping there. Updated POM.xml.
With the new version the Plugin site will take docs from GitHub

**Comment by olivergondza** [comment]
Thanks, was not aware of the WEBSITE-406 effort.

**Comment by oleg-nenashev** [comment]
FTR https://groups.google.com/forum/#!topic/jenkinsci-dev/VSdfVMDIW-A  . I was just announced 2 days ago, and I forgot to link it in my last comment

**Comment by tiger5226** [comment]
Its also probably a good idea to include the format of the junit xml files https://llg.cubic.org/docs/junit/

**Comment by oleg-nenashev** [comment]
@olivergondza would it be possible to merge it so that the improvements get submitted as follow-ups?

**Pull request merged**